_layouts/sidebar.md

```diff
@@ -16,4 +16,5 @@
     - [Sambhav Gupta](../journeys/sambhavgupta0705.md)
     - [Shourjjo Majumder](../journeys/shourgamer2.md)
     - [Vikas Ganiga](../journeys/vikasganiga05.md)
+   
```

Comment:
```suggestion
```

Delete unnecessary line.

It might mess up the gh-action if this line is inculded